types/diffable-html/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for diffable-html 5.0
+// Project: https://github.com/rayrutjes/diffable-html#readme
+// Definitions by: Zhang Yi Jiang <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function format(
+    html: string,
+    options?: {
+        sortAttributes?: (attrs: string[]) => string[];
```

Comment:
```suggestion
        sortAttributes?: (names: string[]) => string[];
```
https://github.com/algolia/diffable-html/blob/master/index.js#L23
https://github.com/algolia/diffable-html/blob/master/__tests__/index.js#L275
The implementer has the last saying, but let's keep this aligned with JS